konnectivity-socks5-proxy/http_proxy.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"bufio"
+	"encoding/base64"
+	"errors"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"golang.org/x/net/proxy"
+)
+
+func init() {
+	// The proxy package only interprets the ALL_PROXY variable
+	os.Setenv("ALL_PROXY", os.Getenv("HTTP_PROXY"))
```

Comment:
does this mean that the HTTPS_PROXY env var is ignored?

Reply:
By the proxy package which we use to get a connection and respecting any local proxy config along the way, yes

Reply:
as mentioned on Slack, pls default to use the HTTPS_PROXY value